curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -1405,9 +1408,10 @@ void DataCache::Write(uint64_t chunkPos, uint64_t len, const char *data,
 }
 
 void DataCache::Release() {
-    chunkCacheManager_->ReleaseReadDataCache(chunkPos_);
-
-    return;
+    auto chunkCacheManager = chunkCacheManager_.lock();
```

Comment:
do not need. 
Release()  is now only called by ReadCacheReleaseExecutor. We can get chunkCacheManager in FsCacheManager.
Datacahe does not require reverse dependency on chunkCacheManager.

Reply:
this change has been removed because the case that can cause chunkCacheManager_ expired has been implicitly fix by @baijiaruo